core/src/test/java/io/lionweb/language/LanguageTest.java

```diff
@@ -20,4 +21,24 @@ public void addDependency() {
     m1.addDependency(m3);
     assertEquals(Arrays.asList(m2, m3), m1.dependsOn());
   }
+
+  @Test
+  public void getAnnotationByName() {
+    Language m1 = new Language("m1");
```

Comment:
Minor thing: why are these languages identified as `m1`, etc., instead of `l1`, etc. (or even `lang1`, etc.)?

Reply:
Right! It is because a long time ago Language was called Metamodel, I think. At the time of the first poc. I will change that.